README.Rmd

```diff
@@ -23,14 +23,57 @@ A shiny app for automated detection of signals in infectious disease surveillanc
 This app allows the user to investigate generated signals and corresponding surveillance time series. It includes upload of csv or Excel files, data checks, possibility to set parameters for signal detection, visualisation of results and generation of reports. The tool is interactive and flexible so that it can be used in different contexts and customised to the user’s needs.
 
 ## Installation
-
 You can install the development version of SignalDetectionTool from [GitHub](https://github.com/United4Surveillance/signal-detection-tool) with:
 
 ``` r
 # install.packages("devtools")
 devtools::install_github("United4Surveillance/signal-detection-tool")
 ```
-
+<details>
+  <summary><strong>Installing latest SignalDetectionTool version</strong></summary>
+  
+  <ul>
+    <li>You can download the latest release of the SignalDetectionTool through the Releases page</li>
+    <li>To navigate to the Release page, scroll up and look on the right-hand side of the page under a section labelled Releases. Click on 'Releases'</li>
+    <li>You can find the latest tool release at the top of the page</li>
+    <li>There are three possible ways to install the tool:</li>
+    <ul>
+      <li>If you just want to run the tool and do not want or need to see the code behind, you can follow the installation using .tar.gz</li>
+      <li>If you want to see the code behind the app in addition to running the tool, you can follow the installation from source after unzipping the zip file</li>
+      <li>If you want to explore the code and work with it, you can best use the source code (zip)</li>
+    </ul>
+  </ul>
+
+  <strong>Installation using .tar.gz</strong>
+  <ul>
+    <li>Download the provided .tar.gz file and save it on your computer</li>
+    <li>Install the package <code>remotes</code> by putting <span style="color:blue;">install.package("remotes")</span> in the console</li>
+    <li>Install the package dependencies for the SignalDetectionTool using the full path where you stored the .tar.gz file and replace the x.x.x with the current version of the binary. For example: <span style="color:blue;">remotes::install_deps("C:/Users/YourUsername/Downloads/SignalDetectionTool_x.x.x.tar.gz")</span></li>
+    <li>Execute this command in the console</li>
+    <li>This should also install any dependencies you need to run the tool. Wait until the installation finishes successfully</li>
+    <li>Write the following command into the console to install the tool: <span style="color:blue;">install.packages("path_to_the_tool/SignalDetectionTool_x.x.x.tar.gz", repos = NULL, type="source")</span></li>
+    <li>Continue to read 'Running the shiny application'</li>
+  </ul>
+
+
```

Comment:
Here should be the installation instructions for the binary now. SignalDetectionTool_x.x.x.zip: